src/action-creators/cursorDown.js

```diff
@@ -8,17 +8,17 @@ import {
   isDivider,
   restoreSelection,
   selectNextEditable,
-  getThoughtsRanked,
+  getThoughts
 } from '../util.js'
 
 export const cursorDown = ({ target }) => dispatch => {
   const { cursor } = store.getState()
   if (cursor) {
     // select next editable
     const nextThought = getThoughtAfter(cursor)
-    const descendants = getThoughtsRanked(cursor)
+    const children = getThoughts(cursor)
     if (nextThought) {
-      const nextThoughtsRanked = descendants && descendants.length > 0 ? contextOf(cursor).concat(cursor).concat(descendants[0]) : contextOf(cursor).concat(nextThought)
+      const nextThoughtsRanked = children && children.length > 0 ? contextOf(cursor).concat(cursor).concat(children[0]) : contextOf(cursor).concat(nextThought)
```

Comment:
You still have `contextOf(cursor).concat(cursor).concat(children[0])` which we know results in invalid values. Does `cursor.concat(children[0])` work? If not, can you describe what you are try are trying to accomplish, then we can perhaps figure it out together?